components/QueueTable.js

```diff
@@ -92,6 +92,57 @@ const ShowUploadQueue = (props) => {
         </a>
       ),
     },
+    {
+      id: "userName",
+      label: "Username",
+      minWidth: 150,
+      align: "left",
+      format: (value) => (
+        <a
+          className={classes.id}
+          href={"https://meta.wikimedia.org/wiki/" + value}
+          target="_blank"
+        >
+          {value}
+        </a>
+      ),
+    },
+    {
+      id: "date",
+      label: "Date uploaded",
+      minWidth: 150,
+      align: "center",
+      format: (value) => {
+        const months = [
+          "January",
+          "February",
+          "March",
+          "April",
+          "May",
+          "June",
+          "July",
+          "August",
+          "September",
+          "October",
+          "November",
+          "December",
+        ];
+        return (
+          value.split("-")[0] +
+          " " +
+          months[value.split("-")[1]] +
+          " " +
+          value.split("-")[2]
+        );
+      },
+    },
+    {
```

Comment:
No need to divide Date and Time into separate columns. We can just show the "Timestamp" column in the following date format: YYYY-MM-DD HH:MM (UTC)